apps/dashboard/app/controllers/files_controller.rb

```diff
@@ -175,6 +175,9 @@ def parse_path(path = params[:filepath], filesystem = params[:fs])
     if filesystem == 'fs'
       @path = PosixFile.new(normal_path)
       @filesystem = 'fs'
+    elsif filesystem == 'navigate'
+        @path = PosixFile.new(normal_path)
+        @filesystem = 'navigate'
```

Comment:
Instead of having this toggle, can we just use the regular `/fs/` API?

It costs almost nothing to just render both
```
  Rendered collection of files/_navigate_breadcrumb.html.erb [ times] (Duration: 0.4ms | Allocations: 387)
  Rendered collection of files/_breadcrumb.html.erb [ times] (Duration: 0.4ms | Allocations: 455)
```

Reply:
I tried. I couldn't get it to work properly using /fs.  I'll take another look at it.

Reply:
was less problematic to add these 3 lines. I am going this route.

Reply:
What? I'm not sure I follow the problems. I got it to work with this diff. Toggles mean confusion when we go to debug stuff.

[diff.txt](https://github.com/OSC/ondemand/files/11040955/diff.txt)

Reply:
@johrstrom   You are better than I.  I patched it, and it worked. Thanks for the help. :)